canonical/cloud-init
# Add CloudCIX to the data source list.

## Proposed Commit Message
```
feat(packaging): Add CloudCIX to data source list

This commit adds the recently merged CloudCIX data source to the data
sources in the Ubuntu package.
```

## Additional Context
https://github.com/canonical/cloud-init/pull/1351

## Test Steps
1. Build cloud-init package with this change in place.
2. Install package with default configuration.
3. Check  if `CloudCIX` is present in the list of data sources in `/etc/cloud/cloud.cfg.d/90_dpkg.cfg`.

## Merge type
